README.md

```diff
@@ -119,86 +114,83 @@ templates:
 
 Key | Type | Default | Definition
 ---|--- | ---|---
-[fields](#fields) | hash | nil | list of fields that make this part's form
+[fields](#fields) | hash | `nil` | list of fields that make this part's form
 position | integer | auto-increment | position of the part on the admin page
-repeatable | boolean | false | can the client add multiple of these parts (example a slide in a slideshow)
+repeatable | boolean or string | `false` | can the client add multiple of these parts (example a slide in a slideshow), if a string is passed, `true` is assumed and the string becomes the label for a single part
 
-
-####fields
+#### Fields
 Each field sub-section starts with the name of it's unique identifier. This name is used to reference the field in the code an thus should not change throughout the life of the project.
 
 Key | Type | Default | Definition
 ---|--- | ---|---
 label | string | identifier | form field label
-hint | string | nil | form field hint
-placeholder | string | nil | form field placeholder
+hint | string | `nil` | form field hint
+placeholder | string | `nil` | form field placeholder
 position | integer | auto-increment | position of the field within the part (only for the admin area)
-required | boolean | true | must this field have a value for the form to save
-type | string | `string` | choose between: `boolean`, `date`, `decimal`, `file`, `image`, `integer`, `rich_text`, `text`, `string`, `resource_link` (needs value: resource_linked (same as the controller name ex: pianos)), `external_link` (needs value: http://site.com)
-default | string | nil | set the value but don't overwrite (only set if nil). Put default images in /assets/images/defaults and default files in /assets/files/defaults
-value | string | nil | USE ONLY FOR types `resource_link` and `external_link` (this will overwrite user input, in most cases use the `default` key instead)
-
+required | boolean | `true` | must this field have a value for the form to save
+type | string | `'string'` | choose between: `'boolean'`, `'date'`, `'decimal'`, `'file'`, `'image'`, `'integer'`, `'rich_text'`, `'text'`, `'string'`, `'resource_link'` (needs value: resource_linked, same as the controller name ex: `'pianos'`), `'external_link'` (needs value: `'http://site.com'`)
+default | string | `nil` | set the value but don't overwrite (only set if `nil`). Put default images in _/assets/images/defaults_ and default files in _/assets/files/defaults_
+value | string | `nil` | USE ONLY FOR types `'resource_link'` and `'external_link'` (this will overwrite user input, in most cases use the **_default_** key instead)
 
 ## Create Resources
 To add an admin area for a model simply type (make sure the model exists first and migrations have been run)
-
 ```sh
-rails g cardboard:resource model_name
+rails generate cardboard:resource model_name
```

Comment:
Thank you. Too many times have I come back to this readme to remember the name of a generator task, cmd+f `generate`, and come up empty handed.